src/main/java/org/kitteh/irc/client/library/implementation/InternalClient.java

```diff
@@ -62,6 +63,9 @@
     @Nonnull
     abstract String getRequestedNick();
 
+    @Nonnull
+    abstract InteractionMap getInteractionDefaults();
```

Comment:
Why not make a getter and setter for this publicly available? Allow for later changes to defaults, like if a bot has a config change, so you don't have to restart just to change.